python/mxnet/initializer.py

```diff
@@ -438,25 +462,32 @@ class FusedRNN(Initializer):
     bidirectional : bool
         should be the same with arguments passed to FusedRNNCell.
     """
-    def __init__(self, init, num_hidden, num_layers, mode, bidirectional=False):
+    def __init__(self, init, num_hidden, num_layers, forget_bias, mode, bidirectional=False):
```

Comment:
actually this breaks loading of previously saved symbols. forget_bias needs to be the last argument with a default value.